src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenThrowTests.cs

```diff
@@ -314,5 +314,92 @@ static void Main()
 }";
             CompileAndVerify(source, expectedOutput: "A");
         }
+
+        [Fact]
+        public void TestThrowConvertedValue()
```

Comment:
This is a deviation from the current C# spec. Clearly you're intending it, but have we taken the change through LDM?

Reply:
Yes.  Actually, it was approved long ago (C# 5 era I think, and already documented this way in the ECMA specification by our agreement) but pending implementation, but given target-typed switch expression it would be a breaking change to do it any later than now.

---
In reply to: [301711198](https://github.com/dotnet/roslyn/pull/37052#discussion_r301711198) [](ancestors = 301711198)

Reply:
Just to confirm, `throw default` is still (properly) disallowed, even though it converts to `Exception`, correct?

---
In reply to: [301744754](https://github.com/dotnet/roslyn/pull/37052#discussion_r301744754) [](ancestors = 301744754,301711198)